hs_core/forms.py

```diff
@@ -980,8 +1133,8 @@ def clean(self):
 
 
 def get_crispy_form_fields(field_names, file_type=False):
-    """
-    This creates a list of objects of type Field
+    """Return a list of objects of type Field.
+
```

Comment:
@aphelionz Retract my previous comment - perhaps making ```"""``` a separate line is clearer.